src/Microsoft.Azure.WebJobs/ConnectionProviderAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
```

Comment:
Hmmm, there are lots of other types that are currently in the root namespace but fall into this same category.  If we want to introduce a new namespace strategy, we should do that in a separate PR and examine all the new recently added types and move as necessary (though many would likely be breaking).

E.g. IAttributeInvokeDescriptor, ODataFilterResolutionPolicy, etc.